externs/ima.js

```diff
@@ -108,19 +108,93 @@ google.ima.AdsManager = class {
 
   /** @override */
   dispatchEvent() {}
+
+  /**
+   * @param {!google.ima.AdsRenderingSettings} adsRenderingSettings
+   */
+  updateAdsRenderingSettings(adsRenderingSettings) {}
 };
 
 
 /** @const */
 google.ima.AdsManagerLoadedEvent = class extends Event {
   /**
    * @param {!HTMLElement} video
+   * @param {!google.ima.AdsRenderingSettings=} adsRenderingSettings
```

Comment:
I'm... actually not sure if you need to define the `google.ima.AdsRenderingSettings` type in the externs?

At no point in the code do we actually access the inner variables of the type, we just take it in as a parameter and then pass it unmodified to the IMA SDK. We also never make an `google.ima.AdsRenderingSettings` object in-code.

So it might just be enough to say that this is an `Object=`. We do stuff like for other pass-through config objects in other externs, like in `externs/mux.js` (for the constructor of `muxjs.mp4.Transmuxer`).

Reply:
@theodab I think that adding the definition is not bad, and it can help people in the integration. I would not take away the work that has already been done.